### PR TITLE
feat: upgrade to holoflows-kit 0.9.0

### PR DESCRIPTION
**Pull request by Jack-Works**
closes #

### CODE REVIEW

File: packages/dashboard/package.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
     "build": "build -- snowpack build"
   },
   "dependencies": {
-    "@dimensiondev/holoflows-kit": "=0.8.0-20210429202414-74fb9a1",
+    "@dimensiondev/holoflows-kit": "^0.9.0-20210726055325-dfbe7fd",
```
Comment (septs) [inline packages/dashboard/package.json:14]:
```suggestion
    "@dimensiondev/holoflows-kit": "=0.9.0-20210726055325-dfbe7fd",
```

File: packages/maskbook/package.json [hunk 2]
```diff
@@ -8,7 +8,7 @@
   "dependencies": {
     "@balancer-labs/sor": "^1.0.0",
     "@dimensiondev/common-protocols": "1.6.0-20201027083702-d0ae6e2",
-    "@dimensiondev/holoflows-kit": "=0.8.0-20210429202414-74fb9a1",
+    "@dimensiondev/holoflows-kit": "^0.9.0-20210726055325-dfbe7fd",
```
Comment (septs) [inline packages/maskbook/package.json:11]:
```suggestion
    "@dimensiondev/holoflows-kit": "=0.9.0-20210726055325-dfbe7fd",
```

File: packages/plugin-infra/package.json [hunk 3]
```diff
@@ -6,7 +6,7 @@
   "types": "./dist/",
   "scripts": {},
   "dependencies": {
-    "@dimensiondev/holoflows-kit": "=0.8.0-20210429202414-74fb9a1",
+    "@dimensiondev/holoflows-kit": "^0.9.0-20210726055325-dfbe7fd",
```
Comment (septs) [inline packages/plugin-infra/package.json:9]:
```suggestion
    "@dimensiondev/holoflows-kit": "=0.9.0-20210726055325-dfbe7fd",
```

File: packages/shared/package.json [hunk 4]
```diff
@@ -3,10 +3,10 @@
   "main": "./dist/index.js",
   "types": "./dist/index.d.ts",
   "dependencies": {
-    "@dimensiondev/holoflows-kit": "=0.8.0-20210429202414-74fb9a1",
+    "@dimensiondev/holoflows-kit": "^0.9.0-20210726055325-dfbe7fd",
```
Comment (septs) [inline packages/shared/package.json:6]:
```suggestion
    "@dimensiondev/holoflows-kit": "=0.9.0-20210726055325-dfbe7fd",
```

File: packages/dashboard/package.json (hunk 1 above)
Comment (septs) [reply packages/dashboard/package.json:14]: move to workspace root?

Comment (Jack-Works) [reply packages/dashboard/package.json:14]: No, it contains some web-extension only thing

**Review by Himself65 — APPROVED** [review]

**Review by septs — APPROVED** [review]

**Pull request merged**